.travis.yml

```
@@ -35,12 +35,11 @@ jobs:
       env: SDK=python
       cache: false
       language: python
-      install:
-        - "pip install awscli"
+      install: skip
       before_script:
-        - "aws s3 cp s3://optimizely-travisci-artifacts/ci/trigger_fullstack-sdk-compat.sh ci/ && chmod u+x ci/trigger_fullstack-sdk-compat.sh"
+        - mkdir travis-ci-tools && pushd travis-ci-tools && git init && git pull https://$CI_USER_TOKEN@github.com/optimizely/travisci-tools.git master && popd
```

```suggestion
        - mkdir $HOME/travisci-tools && pushd $HOME/travisci-tools && git init && git pull https://$CI_USER_TOKEN@github.com/optimizely/travisci-tools.git && popd
```
for clarity, let's put the directory under $HOME/travisci-tools instead of $TRAVIS_BUILD_DIR because this is also the location where the python-sdk repo is.